build.sh

```diff
@@ -69,6 +69,7 @@ start_postgresql
 
 # Run the tests.
 bazel test //... \
+  --cache_test_results=no \
```

Comment:
This needs to be changed back before it is unwipped

Reply:
I think we will have other PRs to roll this out, but if no, for sure :)
I added this to trigger full test builds for measurement purposes.